guide/installation-procedure.md

```diff
@@ -137,6 +137,14 @@ or
 gdebi webots_{{ webots.version.major }}.{{ webots.version.minor }}.{{ webots.version.bugfix }}_amd64.deb
 ```
 
+#### Server edition
+
+Webots requires some graphical features that are usually not available by default on a linux server edition and additional packages need to be available to make it work.
+
+Webots depends on `xserver-xorg-core` and `libxslt1.1`. These packages are automatically installed when using the DEB package, but in case of the tarball package the user has to manually install them.
```

Comment:
I don't understand why you mention `libxslt1.1` and not all the other dependencies... Can't we include libxslt1.1 also in the tarball, like other dependencies? Why shall we make a special case for this one and not the others?